src/main/java/com/squareup/javapoet/MethodSpec.java

```diff
@@ -187,10 +187,6 @@ public Builder addParameter(ParameterSpec parameterSpec) {
       return this;
     }
 
-    public Builder addParameter(Type type, String name, Modifier... modifiers) {
```

Comment:
Good question. Foolish consistency? I don't like how there's a bunch of different mechanisms to do the same thing: `ParameterSpec.of`, 
`Parameter.builder`, and this.

I want to clear out everything that I don't immediately need, and then add em back as necessary.

In this case, its tempting to keep this method & drop ParamaterSpec.of, which are mutually redundant.

Like that better?